### DISCUSSION TITLE
我是怎么写博客的

### DISCUSSION THREAD

**Discussion by giscus** [loveminimal/comment#10] [General]
# 我是怎么写博客的

我是怎么写博客的 - https://walkssi.com/%E6%88%91%E6%98%AF%E6%80%8E%E4%B9%88%E5%86%99%E5%8D%9A%E5%AE%A2%E7%9A%84/

https://walkssi.com/%E6%88%91%E6%98%AF%E6%80%8E%E4%B9%88%E5%86%99%E5%8D%9A%E5%AE%A2%E7%9A%84/

**Comment by Godfrey1024** [comment]
牛！

**Reply by Godfrey1024** [reply]
fdf

**Comment by GraceZhuuu** [comment]
很强，准备搭建个人博客（虽然没啥内容），一直在挑选模板，准备用您的模板了，感谢奉献。

**Reply by loveminimal** [reply]
客气~
